docs: Add contribution rules (#250)
makinacorpus/DbToolsBundle: docs/content/contribute/contribute.md

* Report bugs you find: [Reporting issues](https://github.com/makinacorpus/DbToolsBundle/issues) is essential to help us improve this tool.
  Feel confident enough to correct it? PRs are welcome!
* Want to add more Anonymizers? [Why not distribute your own pack](./pack)?
* Want to add missing functionnality? Create an [issue](https://github.com/makinacorpus/DbToolsBundle/issues)!
* Want to start developing ? Look at ["good first issues"](https://github.com/makinacorpus/DbToolsBundle/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)

## Contribution rules

While we want DbToolsBundle to be a community project, **we do not accept any kind of contribution**.

We ask our contributors to follow these simple rules:

### Basic courtesy

Each issue and PR must be introduced with a clear description that expose the problem it solves or the feature
it implements.

_Each wild PR that does not comply with this rule won't be read_

### PR without associated issue

> Note that we accept PR which are not relative to an existing issue (as long as it complies with the previous rule).

Reviewer: ```suggestion
Note that we accept PRs which are not relative to an existing issue (as long as it complies with the previous rule).
```

But we encourage you not do it. Proposing a PR without discussing the problem/the feature with maintainers can
lead to useless work.

### Split your contributions

> It is much easier for mainteners to review small PRs that solve one and only one issue.

Reviewer: ```suggestion
It is much easier for maintainers to review small PRs that solve one and only one issue.
```

- Do not correct multiple issues in the same ticket (unless discussed and approved in advance).
- Do not make changes that are too invasive or too large (unless discussed and approved in advance).

### Mind the documentation

We care a lot about documentation and changelog.

We consider that a feature which is not documented does not exist. So if you add a new
feature, document it in the correct section of our documentation.

Whether you are fixing a bug or developing a new feature: mention it in the `CHANGELOG.md` file.

This file already contains a lot of content, you should quickly understand the expected syntax.

### Ask for help

If you encounter problem while contributing, we encourage you to ask us for help.

> Don't hesitate to push draft PRs and to ping mainteners!

Reviewer: ```suggestion
Don't hesitate to push draft PRs and to ping maintainers!
```

### Ok for AI, but in good intelligence

> While we are no big fan of AI, we accept our contributor to use it.

Reviewer: ```suggestion
While we are no big fans of AI, we accept our contributors to use it.
```

* Talk about it: share this bundle to make it more visible
* Help us with this documentation, you have seen a typo? something is missing? [Issues](https://github.com/makinacorpus/DbToolsBundle/issues) & PRs are welcome (but before, please, take look at our contribution rules below)
* Report bugs you find: [Reporting issues](https://github.com/makinacorpus/DbToolsBundle/issues) is essential to help us improve this tool.
  Feel confident enough to correct it? PRs are welcome!
* Want to add more Anonymizers? [Why not distribute your own pack](./pack)?
* Want to add missing functionnality? Create an [issue](https://github.com/makinacorpus/DbToolsBundle/issues)!
* Want to start developing ? Look at ["good first issues"](https://github.com/makinacorpus/DbToolsBundle/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)

## Contribution rules

While we want DbToolsBundle to be a community project, **we do not accept any kind of contribution**.

We ask our contributors to follow these simple rules:

### Basic courtesy

Each issue and PR must be introduced with a clear description that expose the problem it solves or the feature
it implements.

> _Each wild PR that does not comply with this rule won't be read_

Reviewer: ```suggestion
_Each wild PR that does not comply with this rule won't be read._
```

### PR without associated issue

Note that we accept PR which are not relative to an existing issue (as long as it complies with the previous rule).

> But we encourage you not do it. Proposing a PR without discussing the problem/the feature with maintainers can

Reviewer: ```suggestion
But we encourage you to not do it. Proposing a PR without discussing the problem/feature with maintainers can
```

Author: no sure about this `to` to add, I removed the second `the`

lead to useless work.

### Split your contributions

It is much easier for mainteners to review small PRs that solve one and only one issue.

> - Do not correct multiple issues in the same ticket (unless discussed and approved in advance)

Reviewer 2: I would add some dots at the end of lines maybe?

- Do not make changes that are too invasive or too large (unless discussed and approved in advance)

### Mind the documentation

We care a lot about documentation and changelog.

We consider that a feature which is not documented does not exist. So if you add a new
feature, document it in the correct section of our documentation.

> Whether you are fixing a bug or developing a new feature : mention it in the `CHANGELOG.md` file.

Reviewer 2: There is no space between the word and `:` in english (I think).
